code/modules/reagents/reagent_containers/borghydro.dm

```diff
@@ -1,4 +1,4 @@
-#define C2NAMEREAGENT	"[initial(reagent.name)] (Has Side-Effects)"
+#define CHEMEFFECT "[initial(reagent.name)][(reagent in reagent_desc) ? reagent_desc[reagent] : ""] [initial(reagent.harmful) ? "(Has side effects)" : ""]"
```

Comment:
```suggestion
#define CHEMEFFECT "[initial(reagent.name)][reagent_desc[reagent]] [initial(reagent.harmful) ? "(Has side effects)" : ""]"
```
Same effect, if it's not defined in the associative list it will resolve to `null`, and that's just the same as `""`